node/src/blockchain/blockchain_interface/blockchain_interface_web3/rpc_helpers_web3.rs

```diff
@@ -0,0 +1,314 @@
+// Copyright (c) 2019, MASQ (https://masq.ai) and/or its affiliates. All rights reserved.
+
+use crate::blockchain::blockchain_interface::rpc_helpers::{
+    RPCHelpers, ResultForBalance, ResultForNonce,
+};
+use crate::blockchain::blockchain_interface::BlockchainError;
+use crate::sub_lib::wallet::Wallet;
+use futures::Future;
+use std::rc::Rc;
+use web3::contract::{Contract, Options};
+use web3::transports::Batch;
+use web3::types::BlockNumber;
+use web3::{BatchTransport, Web3};
+
+pub struct RPCHelpersWeb3<T>
+where
+    T: BatchTransport,
+{
+    web3: Rc<Web3<T>>,
+    // TODO waiting for GH-707 (note: consider to query the balances together with the id)
+    _batch_web3: Rc<Web3<Batch<T>>>,
+    contract: Contract<T>,
+}
+
+impl<T> RPCHelpers for RPCHelpersWeb3<T>
+where
+    T: BatchTransport,
+{
+    fn get_transaction_fee_balance(&self, wallet: &Wallet) -> ResultForBalance {
```

Comment:
Yeah. This is design inherited from Substratum. I don't know...
Maybe you're right but I would like to do it you rather because I don't have that feeling for it. 

It's not important to me, but I quite like these results of multiple methods are like a family, where the root of each is the BlockcahinError on the right side, next to the ok results. At least this was a good idea. I think there are like two levels of these structures where you're complaining basically about the upper one only.

I'm not going to do anything with this in this card though...